.github/workflows/ci.yml

```diff
@@ -25,6 +25,7 @@ env:
   GIT_COMMIT: ${{ github.event.pull_request.head.sha || github.sha }}
   # kludge until we move away from runners
   WAIT_FOR_RUNNERS: false
+  FULL_CI: true
```

Comment:
Changes in this file are just intended to make sure e2e tests run on this PR since they're plausibly effected. Will undo them before merge